Minebot/src/main/java/net/famzangl/minecraft/minebot/ai/path/PlantPathFinder.java

```diff
@@ -81,9 +81,14 @@ protected boolean isAtDesiredHeight(AIHelper aiHelper) {
 		@Override
 		public boolean applyToDelta(WorldWithDelta world) {
 			Item anyPlaceItem = seedFilter.type.items[0];
-			BlockState block = ((IPlantable) anyPlaceItem)
-					.getPlant(null, null);
-			world.setBlock(pos, block);
+			try {
+
+				BlockState block = ((IPlantable) anyPlaceItem) //BlockNamedItem cannot be cast to net.minecraftforge.common.IPlantable
+						.getPlant(null, null);
+				world.setBlock(pos, block);
+			} catch (Exception e) {
```

Comment:
Which exception can occur here? Only catch that specific exception. And for all other cases, at least print the error to the log, so that this can be debugged if Minecraft changes something here.

Reply:
You can't cast BlockStateItem to IPlantable, and that's a... significant change that might require some kind of logic rework. However, plant works just fine right until the end where it tries to stop, then it crashes. So I figured it irrelevant to the actual runnings and just put a try catch around it.

Reply:
Then comment out that code. Because in the current way, if anyone ever debugs it, it will be hard to find out why this line is never executed. catch(Exception) makes maintaining software really hard.

Reply:
I didn't know if it was important for during-planting, I only know that it crashes after-planting, maybe it's necessary ): I commented out one of the intercept somethings at some point aswell :eyes: